po/zh_CN.po

```diff
@@ -133,22 +133,22 @@ msgstr "生成压缩归档......"
 #: ../sos/sosreport.py:1498
 #, python-format
 msgid " %s while finalizing archive"
-msgstr ""
+msgstr "%s存档完成"
```

Comment:
Missing space after `%s` ?

Reply:
I am sorry about this mistakes. I'll recheck and submit again.